bench/qemu-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Runs specified SSHFILE on QEMU guest and then kills buildroot
+
+ROOT=buildroot
+TERMINAL="gnome-terminal"
+
+# command line arguments to script with defaults
+HOST_SH="${1:-wiredtiger-copy.sh}" # file to run on host
```

Comment:
This looks like it is specific to wired tiger right now. What do we need to change to make this generic? We will have several benchmarks to run eventually.

Reply:
yeah I'll fix that along with making it optional

Reply:
fixed now. By default, there is no guest or host script